server/src/test/java/org/elasticsearch/index/termvectors/TermVectorsServiceTests.java

```diff
@@ -78,12 +78,14 @@ public void testTook() throws Exception {
     }
 
     public void testDocFreqs() throws IOException {
+        boolean indexPhrases = randomBoolean();
```

Comment:
can we instead split the test into two tests so that both cases are tested every time?